swift-2048/NumberTileGame.swift

```diff
@@ -164,11 +164,10 @@ class NumberTileGameViewController : UIViewController, GameModelProtocol {
     let (userWon, _) = m.userHasWon()
     if userWon {
       // TODO: alert delegate we won
-      let alertView = UIAlertView()
-      alertView.title = "Victory"
-      alertView.message = "You won!"
-      alertView.addButton(withTitle: "Cancel")
-      alertView.show()
+      let alertView = UIAlertController(title: "Victory", message: "You won!", preferredStyle: .alert)
+      alertView.addAction(UIAlertAction(title: "New Game", style: .default, handler: {action in self.reset()}))
+      alertView.addAction(UIAlertAction(title: "Cancel", style: .cancel, handler: nil))
+      self.present(alertView, animated: true)
       // TODO: At this point we should stall the game until the user taps 'New Game' (which hasn't been implemented yet)
```

Comment:
With this change you should be able to remove this comment

Reply:
done, thanks!